dockerfile

```
@@ -0,0 +1,17 @@
+FROM ubuntu:22.04
+
+# Install package from apt
+RUN sed 's@archive.ubuntu.com@free.nchc.org.tw@' -i /etc/apt/sources.list
+RUN apt-get -y update && apt-get install -y ssh make build-essential git curl
+
+# Install nodejs, check the environment have node and npm.
+RUN curl -sL https://deb.nodesource.com/setup_16.x | bash
+RUN apt-get install -y nodejs
+RUN node -v && npm -v 
```

```suggestion
FROM node:lts
```

How about using [the official Node image](https://hub.docker.com/_/node)?